packages/zoe/test/unitTests/contracts/vpool-xyk-amm/test-amm-governance.js

```diff
@@ -3,195 +3,50 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
-import bundleSource from '@agoric/bundle-source';
 import { E } from '@agoric/eventual-send';
-import { makeLoopback } from '@agoric/captp';
 
-import { resolve as importMetaResolve } from 'import-meta-resolve';
-import { ParamType } from '@agoric/governance';
 import { makeIssuerKit, AmountMath } from '@agoric/ertp';
-import { makeFakeVatAdmin } from '../../../../tools/fakeVatAdmin.js';
+import { CONTRACT_ELECTORATE } from '@agoric/governance/src/paramGovernance/governParam.js';
```

Comment:
deep import. hm.

Reply:
is it?

Changing it isn't critical, but I'm getting mixed signals about what you think you wrote.